frontend/src/views/Camps.vue

```diff
@@ -42,12 +39,12 @@
         </v-list-item>
       </v-list>
       <v-expansion-panels
-        v-if="prototypeCamps.length > 0 || pastCamps.length > 0"
+        v-if="(isAdmin() && prototypeCamps.length > 0) || pastCamps.length > 0"
         multiple
         flat
         accordion
       >
-        <v-expansion-panel v-if="prototypeCamps.length > 0">
+        <v-expansion-panel v-if="isAdmin() && prototypeCamps.length > 0">
```

Comment:
No need to wait for `loading` here?

Reply:
Added with https://github.com/ecamp/ecamp3/pull/3158/commits/82d5e939f1f900b64832d6643e11b62e3f510751